app/views/HasNiphlView.scala.html

```diff
@@ -26,6 +28,12 @@
 
 @layout(pageTitle = title(form, messages("hasNiphl.title"))) {
 
+    <h1 class="govuk-heading-l">@messages("hasNiphl.heading")</h1>
+    <p class="govuk-body">@messages("hasNiphl.p1")</p>
+    <p class="govuk-body">@messages("hasNiphl.p2")
+        <a class="govuk-link" href="https://www.gov.uk/guidance/moving-plants-from-great-britain-to-northern-ireland">@messages("hasNiphl.p2.linkText")</a>.
```

Comment:
We kept the link component on ours, FYI - but this is fine, too. Please comment on marcy's pr if you're not okay with the component being used